src/core/BufferGeometry.js

```diff
@@ -1018,7 +1018,7 @@ THREE.BufferGeometry.prototype = {
 
 				vertexMap[ vid ] = new_vid;
 				revVertexMap[ new_vid ] = vid;
-				sortedIndices[ indexPtr ++ ] = new_vid - offset.index; //XXX overflows at 16bit
+				sortedIndices[indexPtr++] = new_vid - offset.indexOffset; //XXX overflows at 16bit
```

Comment:
My editor gets very annoyed with square brackets; is ok with round brackets but it eats square brackets spaces. So I have to edit then undo to restore formatting - don't always catch it :(